### ISSUE TITLE
Cannot build v0.4.3

### ISSUE THREAD

**Issue by Coppini** [pjedge/longshot#62]
I'm trying to build v0.4.2 and v0.4.3 from the source code, using `cargo install --path .` or `cargo build --release`, but neither command is working. I did install all of the dependencies described in the README before attempting it.

It throws a lot of warning messages, until finally throws an error:
```
error: failed to compile `longshot v0.4.3 (/path/to/longshot)`, intermediate artifacts can be found at `/path/to/longshot/target`

Caused by:
  build failed
```

I already had `longshot==0.4.1` installed on my machine before (installed with conda), so I was unsure if there might be some compatiblity issue, and tried building `longshot==0.4.3` using docker, but had the same error. 

Is there any new dependency missing from the README? If not, then is there any up-to-date docker image or Dockerfile for the newest version, so I could use longshot in docker using that?

**Comment by vibansal** [comment]
There is a version issue with the latest release on conda that causes the errors on some platforms, we are working to resolve it.

**Comment by Coppini** [comment]
Managed to build a Dockerfile that seems to be running fine, in case someone else is having trouble installing the newest version without conda.

```
FROM ubuntu:18.04 AS install-longshot

## Install dependencies
#### you can use either wget or git+ca-certificates, no need to get both
#### musl-tools used to install x86_64-unknown-linux-musl so your binary has no external dependencies
RUN apt-get update && apt-get install -y --no-install-recommends \
    cargo \
    zlib1g-dev \
    xz-utils \
    libclang-dev \
    clang \
    build-essential \
    curl \
    wget \
    ca-certificates \
    git \
    musl-tools \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*

## Install RUST
#### x86_64-unknown-linux-musl is used so the binary won't have external dependencies
RUN curl https://sh.rustup.rs -o install-rustup.sh
RUN chmod +x install-rustup.sh
RUN ./install-rustup.sh -y --target x86_64-unknown-linux-musl
ENV PATH=/root/.cargo/bin:$PATH

## Download longshot
### Fixed version to 0.4.3
RUN wget https://github.com/pjedge/longshot/archive/v0.4.3.tar.gz
RUN tar -xzvf v0.4.3.tar.gz
WORKDIR /longshot-0.4.3

## Clone longshot repository
#### This is an alternative way to the download option. If you use this, comment the download section out.
#### This has no fixed version, and will just get the latest on master
# RUN git clone https://github.com/pjedge/longshot
# WORKDIR /longshot

## Install longshot
RUN cargo build --target x86_64-unknown-linux-musl --release
RUN cargo install --target x86_64-unknown-linux-musl --path .

## Copy the binary to a minimal docker image
FROM alpine:3.12.4
# RUN apk add --no-cache bash # My workflow required a bash entrypoint. Commented it out for a more general use.
COPY --from=install-longshot /root/.cargo/bin/longshot /bin/longshot
```
